starknet_py/net/schemas/common.py

```diff
@@ -95,6 +95,8 @@ def _deserialize(
 
 class TransactionTypeField(fields.Field):
     def _serialize(self, value: Any, attr: str, obj: Any, **kwargs):
+        if value.name == "INVOKE":
+            return "INVOKE_FUNCTION"
         return value.name if value is not None else ""
```

Comment:
Do you know why a check for `None` is here?

Reply:
No, but let's leave it